engine/installation/linux/docker-ce/centos.md

```diff
@@ -0,0 +1,258 @@
+---
+description: Instructions for installing Docker CE on CentOS
+keywords: requirements, apt, installation, centos, rpm, install, uninstall, upgrade, update
+redirect_from:
+- /engine/installation/centos/
+title: Get Docker CE for CentOS
+---
+
+{% assign minor-version = "17.03" %}
+
+To get started with Docker CE on CentOS, make sure you
+[meet the prerequisites](#prerequisites), then
+[install Docker](#install-docker).
+
+## Prerequisites
+
+### Docker EE customers
+
+To install Docker Enterprise Edition (Docker EE), go to
+[Get Docker EE for Ubuntu](/engine/installation/linux/docker-ee/centos/)
+**instead of this topic**.
+
+To learn more about Docker EE, see
+[Docker Enterprise Edition](https://www.docker.com/enterprise-edition/){: target="_blank" class="_" }.
+
+### OS requirements
+
+To install Docker CE, you need the 64-bit version of CentOS 7.
+
+### Uninstall old versions
+
+Older versions of Docker were called `docker` or `docker-engine`. If these are
+installed, uninstall them, along with associated dependencies.
+
+```bash
+$ sudo yum remove docker \
+                  docker-common \
+                  container-selinux \
```

Comment:
heads up: for docker 17.06 we'll be relying on the RHEL SELinux packages   so some may no longer have to be uninstalled